api/nodes.go

```diff
@@ -513,6 +515,41 @@ type NodeReservedNetworkResources struct {
 	ReservedHostPorts string
 }
 
+type CSITopology struct {
```

Comment:
Missing comment or maybe not necessary?

Reply:
Not really necessary in the API package bc it's too easy to get out of sync with nomad/structs/...